run_tests.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+
+# Default values
+NETWORK="regnet"
+LOG_LEVEL="none"
+BACKTRACE=0
+
+# Parse command-line arguments
+while getopts "n:l:b:t:" opt; do
+  case $opt in
+    n)
+      if [ "$OPTARG" == "signet" ] || [ "$OPTARG" == "regnet" ]; then
+        NETWORK=$OPTARG
+      else
+        echo "Invalid network option. Use 'regnet' or 'signet'."
+        exit 1
+      fi
+      ;;
+    l)
+    if [ "$OPTARG" == "debug" ] || [ "$OPTARG" == "info" ] || [ "$OPTARG" == "none" ]; then
+        LOG_LEVEL=$OPTARG
+    else
+        echo "Invalid log level option. Use 'info', 'debug', or 'none'."
+        exit 1
+    fi
+    ;;
+    b)
+      BACKTRACE=$OPTARG
+      ;;
+    t)
+      TEST_NAME="$OPTARG"
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG" >&2
+      exit 1
+      ;;
+    :)
+      echo "Option -$OPTARG requires an argument." >&2
+      exit 1
+      ;;
+  esac
+done
+
+# Copy the bitcoin configuration from the current directory to ~/.bitcoin
+cp ./bitcoin.conf ~/.bitcoin/
```

Comment:
Not sure that's usefull, some people might already have a bitcoin.conf. 
The file is an example.